src/main/java/org/immregistries/vfa/connect/IISConnector.java

```diff
@@ -1327,8 +1327,7 @@ private String sendRequest(String request) throws IOException, NotAuthenticated
     urlConn.setDoInput(true);
     urlConn.setDoOutput(true);
     urlConn.setUseCaches(false);
-    urlConn.setRequestProperty("Content-Type", "text/xml; charset=utf-8");
-    urlConn.setRequestProperty("SOAPAction", "\"http://tempuri.org/ExecuteHL7Message\"");
+    urlConn.setRequestProperty("Content-Type", "application/soap+xml; charset=utf-8");

```

Comment:
Content type should be "application/soap+xml" for SOAP 1.2, as documented here: 
https://www.w3.org/TR/2007/REC-soap12-part2-20070427/#httpmediatype